test/test-examples-lint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# lint the examples using mgmt lint
+
+set -e
+
+# shellcheck disable=SC1091
+. test/util.sh
+
+echo running "$0"
+
+ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${ROOT}"
+
+failures=''
+
+# lint .mcl examples
+for file in $(find examples/lang/ -maxdepth 3 -type f -name '*.mcl'); do
```

Comment:
I'm really pleased you added a test for this, because I was thinking we needed this :)